### PR TITLE
Update scrollview.md adding new prop automaticallyAdjustsScrollIndicatorInsets

### PR DESCRIPTION
**Pull request by justinwh**
Adds the new prop `automaticallyAdjustsScrollIndicatorInsets` to the docs for `ScrollView`.

Prop added here: https://github.com/facebook/react-native/pull/29809

### CODE REVIEW

File: docs/scrollview.md [hunk 1]
```diff
@@ -112,6 +112,16 @@ Controls whether iOS should automatically adjust the content inset for scroll vi
 
 ---
 
+### `automaticallyAdjustsScrollIndicatorInsets` <div class="label ios">iOS</div>
+
+Controls whether iOS should automatically adjust the scroll indicator insets. ([UIScrollView documentation](https://developer.apple.com/documentation/uikit/uiscrollview/3198043-automaticallyadjustsscrollindica))
```
Comment (lunaleaps) [inline docs/scrollview.md:117]: Could I suggest: 
```
...adjust the scroll indicator insets. See Apple's [documentation on the property](https://developer.apple.com/documentation/uikit/uiscrollview/3198043-automaticallyadjustsscrollindica)
```

**Comment by lunaleaps** [comment]
@justinwh Let me know what you think about my suggestion! Otherwise looks good to me!

@Simek what does the tag `Wait on future release` indicate? Let me know if there's a doc explaining this that I'm missing

**Comment by Simek** [comment]
> @Simek what does the tag `Wait on future release` indicate? Let me know if there's a doc explaining this that I'm missing

The description is avaible on the labels list - https://github.com/facebook/react-native-website/labels - shortly, it means that the PR adds a docs change which will be a part of future release, beyond `next` version, which on `0.65` release becomes `0.65` docs.

Also this PR is missing "Waiting on PR" label, since the main repository PR has not been merged yet.

File: docs/scrollview.md (hunk 1 above)
Comment (justinwh) [reply docs/scrollview.md]: Sounds good to me. I'll update the PR.

**Comment by lunaleaps** [comment]
Gotcha! Thanks for explaining! That's right, this would be beyond `next`. Does that inhibit us from merging?

**Review by lunaleaps — APPROVED** [review]

**Pull request merged**